lang/shared/src/main/scala/com/wavesplatform/lang/v1/Repl.scala

```diff
@@ -0,0 +1,37 @@
+package com.wavesplatform.lang.v1
+
+import com.wavesplatform.lang.directives.values.{StdLibVersion, V3}
+import com.wavesplatform.lang.v1.compiler.{ExpressionCompiler}
+import com.wavesplatform.lang.v1.evaluator.EvaluatorV1
+import com.wavesplatform.lang.v1.evaluator.ctx.impl.{CryptoContext, PureContext}
+import com.wavesplatform.lang.v1.parser.Parser
+import monix.execution.atomic.Atomic
+import cats.implicits._
+import com.wavesplatform.lang.v1.compiler.Terms.EVALUATED
+import com.wavesplatform.lang.v1.compiler.Types.{FINAL, UNIT}
+
+case class Repl(ver: StdLibVersion = V3) {
+  private val Global: BaseGlobal = com.wavesplatform.lang.Global
+  private val ctx         = CryptoContext.build(Global, ver) |+| PureContext.build(Global, ver)
```

Comment:
please add wavescontext too, mock Environment to throw exceptions ("function not available in repl context")

Reply:
+